src/patternfly/components/NavSystem/docs/code.md

```diff
@@ -0,0 +1,48 @@
+## Overview
+
+The navigation system relies on several different sub-components: 
+
+* `.pf-nav__list` - Default navigation list. It is the basis for both default and expandable, vertical navigation.
+* `.pf-nav__list-simple` - Nav list simple is a shareable component this is used with grouped navigation and as secondary navigation.
+* `.pf-nav__list-horizontal` - Nav list horizontal is a shareable component that can be used within the page header, as primary navigation, or as tertiary navigation, when expandable, vertical navigation is implemented.
```

Comment:
This seems very similar to secondary nav. This and secondary nav seem similar to a `tab` component. What is the plan for Tabs vs Navs?

Reply:
That's a good point @dmiller9911 . Seems like the thing called tertiary nav in the design mocks is really the same as the component called Secondary Nav which is also just a tab bar. The Horizontal Primary Nav might be a bit different because I think we wanted it to have some unique responsive behaviors.

Reply:
Both good points. Tabbed content and horizontal navigation may look alike, but semantics are important here. Navigation should always be a list of links to unique pages, whereas tabs are showing/hiding content on the same page. I think we need to revisit SecondaryNav and align with https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html for improved accessibility, which would change the markup significantly.
- SecondaryNav should be named something along the lines of TabbedContent/Tablist
- TabbedContent/Tablist component should include corresponding `tabpanel`
- Should be using buttons instead of links, with all the accessibility considerations noted here https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html#rps_label

@mcarrano Horizontal nav and tabbed content are very similar, but maybe we should consider adding some design details to differentiate them?

@jgiardino @andresgalante What do you think?